### DISCUSSION TITLE
Copy and paste limited in 1.3.1

### DISCUSSION THREAD

**Discussion by Paolo-O** [rustdesk/rustdesk#9568] [General]
### Bug Description

When copying and pasting using 1.3.1 it is only possible to paste a text content.
As an example, it is not possible to paste a formula in Excel, but only the number representing its result.
All releases up to 1.3.0 were copying and pasting correctly

### How to Reproduce

As an example, it is not possible to paste a formula in Excel, but only the number representing its result.

### Expected Behavior

The formula should be pasted, not just its result

### Operating system(s) on local side and remote side

Win 11 -> Win 11                                                                          11 ->

### RustDesk Version(s) on local side and remote side

1.3.1 -> 1.3.1

### Screenshots

Not applicable

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9570
